### PR TITLE
Portal Improvements

### PR DESCRIPTION
**Pull request by hadia**
Services that a user is logged into should be shown at the top of the list 

## Issue being fixed or feature implemented

https://user-images.githubusercontent.com/1075310/146955015-10ee6d56-447f-48d6-b66f-88eb3a8185ae.mp4

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [x] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: common/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -765,6 +765,10 @@
         <item name="android:textColor">@color/red_300</item>
     </style>
 
+    <style name="Subtitle2.14">
```
Comment (Syn-McJ) [inline common/src/main/res/values/styles.xml:768]: You don't need to specify any styles with font sizes. Subtitle2 is derived from `TextAppearance.MaterialComponents.Subtitle2` which is already 14sp. Same for all the other design system styles.

File: wallet/res/layout/activity_buy_and_sell_liquid_uphold.xml [hunk 2]
```diff
@@ -1,368 +1,140 @@
...
+        app:layout_anchor="@id/select_a_service_to_buy_and_sell_Dash"
+        app:layout_anchorGravity="bottom"
+        app:layout_behavior="de.schildbach.wallet.ui.CollapsingImageBehavior"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/select_a_service_to_buy_and_sell_Dash">
+        <!--    <include-->
+        <!--        layout="@layout/app_bar_general"-->
+        <!--        android:visibility="gone"-->
+        <!--        android:layout_width="match_parent"-->
+        <!--        android:layout_height="wrap_content" />-->
```
Comment (Syn-McJ) [inline wallet/res/layout/activity_buy_and_sell_liquid_uphold.xml:91]: Commented code - is it still needed?

File: wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt [hunk 3]
```diff
@@ -260,51 +297,45 @@ class BuyAndSellLiquidUpholdActivity : LockScreenActivity() {
...
-            }
-        }
     }
 
     fun setNetworkState(online: Boolean) {
         network_status_container.isVisible = !online
-        liquid_container.isEnabled = online && LiquidConstants.hasValidCredentials()
-        uphold_container.isEnabled = online && UpholdConstants.hasValidCredentials()
+        // liquid_container.isEnabled = online && LiquidConstants.hasValidCredentials()
+        // uphold_container.isEnabled = online && UpholdConstants.hasValidCredentials()
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt:338]: Commented code - better to cleanup or leave an explainer if you still need it

File: wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt [hunk 4]
```diff
@@ -403,76 +390,19 @@ class BuyAndSellLiquidUpholdActivity : LockScreenActivity() {
             for (i in 0 until cryptoArray.length()) {
                 val currency = cryptoArray.getJSONObject(i).getString("currency")
                 if (currency == "DASH") {
-                    liquid_balance_container.visibility = View.VISIBLE
+                    // liquid_balance_container.visibility = View.VISIBLE
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt:393]: Commented code - see above

**Review by Syn-McJ — APPROVED** [review]
Great work, there are some minor notes on styles and commented code that would be nice to fix.

**Review by HashEngineering — COMMENTED** [review]
Approved on the basis that @Syn-McJ 's comments are addressed.

**Pull request closed without merging**